### PR TITLE
Added Controller for awsssmchaosrunner

### PR DESCRIPTION
**Pull request by Debanitrkl**
### What problem does this PR solve?
This is a controller for the integration of awsssmchaosrunner with chaos mesh according to this [rfc.](https://github.com/chaos-mesh/rfcs/pull/16)

### What is changed and how does it work?
Added a file under `controllers/awschaos` called `attack-configuration`

Tests

- [x] Unit test
- [ ] E2E test
- [ ] Manual test (add detailed scripts or steps below)

Related changes

- [ ] Need to update the documentation

### Does this PR introduce a user-facing change?

```release-note
Yes a new feature under `awschaos` is added
```

### CODE REVIEW

File: controllers/awschaos/attack-configuration/types.go [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+			Containers: []v1.Container{
+				{
+					Name: "attackConfiguration",
+					Image: "",
+					Args: { },
+				},
+			},
+		},
+	}
+	e.Log.Info("Running the Container")
```
Comment (STRRL) [inline controllers/awschaos/attack-configuration/types.go:63]: After creating the Pod, you should update the status of AwsChaos. It means you should also create new fields to describe the status of that created Pod.

And that status will be also used when `Recover()`.

Comment (Debanitrkl) [reply controllers/awschaos/attack-configuration/types.go:63]: Yeah working on that

**Comment by STRRL** [comment]
I am going to close this PR because it seems not to be updated

**Comment by Debanitrkl** [comment]
Can we put this in WIP I'm on a break and I will get back to this as soon as possible. 
Sorry for the delay got caught up with lot of work in last two months.

**Pull request closed without merging**